code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -8,12 +8,12 @@
 	meat = /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/zombie
 	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOBLOOD,RADIMMUNE,NOZOMBIE,EASYDISMEMBER,EASYLIMBATTACHMENT)
 	mutant_organs = list(/obj/item/organ/tongue/zombie)
+	
 
 /datum/species/zombie/infectious
 	name = "Infectious Zombie"
 	id = "memezombies"
 	limbs_id = "zombie"
-	no_equip = list(slot_wear_mask, slot_head)

```

Comment:
You might have a point about teamkills, I mean they'll quickly realize their error but maybe I'll come up with an answer.

On the other hand people can try to avoid zombies by walking and acting like them.... hmm. Emergent gameplay might carry the day here.